styles.css

```diff
@@ -0,0 +1,26 @@
+body{
+   
```

Comment:
It's better for the convention to let a space between the word and the "{".
It would be better not to have a white line